content/en/_index.html

```diff
@@ -43,7 +43,7 @@ <h2>The Challenges of Migrating 150+ Microservices to Kubernetes</h2>
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Watch Video</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2022/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu22" button id="desktopKCButton">Attend KubeCon Europe on May 17-20, 2022</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2023/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu23" button id="desktopKCButton">Attend KubeCon Europe on April 17-21, 2023</a>
```

Comment:
```suggestion
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2023/" button id="desktopKCButton">Attend KubeCon Europe on April 17-21, 2023</a>
```
? Copied from the previous link, no clue if valid

Reply:
We should link to the next KubeCon which according to https://events.linuxfoundation.org/about/calendar/?_sft_lfevent-category=kubecon-and-other-cncf-events is KubeCon 2022 North America.

Reply:
It's linked on line 51 below, or should the positions be swapped?

Reply:
Yep, we order them by date.
Maybe one day we can automate these updates. They aren't frequent, though.